tests/pipelines/test_pipelines_audio_classification.py

```diff
@@ -31,9 +31,9 @@
 
 
 @is_pipeline_test
-@require_torch
```

Comment:
to allow to run `test_modeling_tf_wav2vec2.py -k "test_pipeline_audio_classification"`